bitnami/elasticsearch/templates/master-statefulset.yaml

```diff
@@ -14,6 +14,7 @@ spec:
     matchLabels: {{- include "elasticsearch.matchLabels" . | nindent 6 }}
       role: master
   serviceName: {{ template "elasticsearch.master.fullname" . }}
+  podManagementPolicy: Parallel
```

Comment:
Is this necessary exclusively for master nodes? What happens with data, ingest or coordinating-only nodes?

Reply:
There is no interdependency between the rest of roles in the Elasticsearch cluster, the only dependency is with the master nodes.

So this is only required for the master nodes and irrelevant for the rest, in terms of setting up the cluster.